stripe/src/main/java/com/stripe/android/view/LockableHorizontalScrollView.java

```diff
@@ -27,12 +28,29 @@ public LockableHorizontalScrollView(Context context, AttributeSet attrs, int def
         super(context, attrs, defStyleAttr);
     }
 
+    /**
+     * @return {@code true} if the view is scrollable, otherwise {@code false}.
+     */
     public boolean isScrollable() {
         return mScrollable;
     }
 
+    /**
+     * Sets whether or not the control is scrollable.
+     *
+     * @param scrollable {@code true} if it should be scrollable, or {@code false} if not
+     */
     public void setScrollable(boolean scrollable) {
         mScrollable = scrollable;
+        setSmoothScrollingEnabled(scrollable);
+    }
+
+    @Override
+    public void scrollTo(int x, int y) {
```

Comment:
1. I'm not sure why this is the method that needs to be overridden to block the keyboard scroll from happening.
2. This makes logical sense anyway; if this isn't scrollable, you shouldn't be able to call scrollTo on it.
3. Absolutely nothing I could do in automation triggered this, ever.
4. Without this, I could 100% repro the bug in the simulator.